orion-ui/src/pages/TaskRun.vue

```diff
@@ -13,6 +13,17 @@
         <TaskRunLogs :task-run="taskRun" />
       </template>
 
+      <template v-if="taskRun" #task-inputs-heading>
+        Task inputs
+        <ExtraInfoModal title="Task Inputs">
```

Comment:
I think we need a little spacing here. Icon is a little close

Reply:
Sorry!  The screenshot I provided was an old one - we added padding as part of the component in orion design. It now looks like this:

<img width="573" alt="image" src="https://user-images.githubusercontent.com/40272060/217365997-a7a06f12-91d2-4c96-aff5-ad748b90908d.png">

Reply:
Nice!